### ISSUE TITLE
Replicates menu bar when toggled

### ISSUE THREAD

**Issue by Matthewspear** [nylas/N1#502]
I open Nylas, (Version 0.3.25-c174641 (0.3.25-c174641)) then go to setting and toggle / un-toggle the "show icon in menu bar" option with each toggle creating another instance of the menu bar.

<img width="400" alt="screen 20shot 202015-11-26 20at 2011 00 26 20am" src="https://cloud.githubusercontent.com/assets/1668863/11451878/aec1d066-95cc-11e5-9b90-61ea84c36eee.png">

Feel free to ask me questions!

Thanks,
Matt

**Comment by jstejada** [comment]
wow @Matthewspear , thanks for reporting this! I can't seem to be able to reproduce this, are you sure you are on 0.3.25?

**Comment by Matthewspear** [comment]
Yeah I've check for updates:
<img width="437" alt="screen shot 2015-11-28 at 3 25 14 pm" src="https://cloud.githubusercontent.com/assets/1668863/11452568/4ca9e324-95e4-11e5-9fc1-9b867dc2ac7f.png">

**Comment by jstejada** [comment]
hmm ok thanks! is this behavior always consistent? what version of mac are you using?

**Comment by Matthewspear** [comment]
El Capitan... that could be the problem:

<img width="291" alt="screen shot 2015-11-28 at 3 29 00 pm" src="https://cloud.githubusercontent.com/assets/1668863/11452577/d47c575a-95e4-11e5-840a-f964c7319236.png">

**Comment by jstejada** [comment]
nope, I'm also running 10.11.1 :disappointed:. So does this always happen to you? Do you have any other details of how you are making it replicate?

**Comment by Matthewspear** [comment]
Yeah well it's only been a recent update the menu bar was added, so since that's been a feature it been a problem - It's not an obvious bug, but seems to be creating a new instance of the menu bar each time its toggled on, I might jump in and look at the code - sadly havn't used much javascript before so I'm not sure how useful I can be!

**Comment by jstejada** [comment]
all right, thanks @Matthewspear I'll dig into it and report back soon

**Comment by bengotow** [comment]
Hey @Matthewspear — I've tried reproducing this with Juan and we haven't had any luck yet. Can you look in Console.app when multiple items start appearing and see if there are any relevant messages there? Mac OS X may be getting confused somehow..

**Comment by Matthewspear** [comment]
@bengotow sorry for the slow reply, had a quick look - not sure if this is relevant but the only message that appeared while I was toggling the button:
```
05/12/2015 11:05:40.561 am ReportCrash[33209]: com.apple.message.domain: com.apple.crashreporter.writereport.crash
com.apple.message.signature: LoginHelper
com.apple.message.signature2: UNBUNDLED ||| 1.0 (1)
com.apple.message.signature3: D54A6A90FCFB9AC2A5564B742D445D47
com.apple.message.result: NO
com.apple.message.summarize: YES
SenderMachUUID: E5ABFA19-9909-3889-B2A1-B9EA272CE592
```

**Comment by grinich** [comment]
@Matthewspear we haven't been able to reproduce this and haven't heard about it from any other users. Are you still seeing this issue?

**Comment by Matthewspear** [comment]
I am still seeing this issue on the latest version - Version 0.3.36-e8239a9 (0.3.36-e8239a9)
<img width="285" alt="screen shot 2016-01-07 at 11 57 14 pm" src="https://cloud.githubusercontent.com/assets/1668863/12186416/723344a2-b59a-11e5-8eb2-5394b4bf5a95.png">

**Comment by bengotow** [comment]
Hey folks! This has been addressed starting in 04.10. We moved management of the system tray to the applications main process so reloading / restating the main window process doesn't cause duplicate items.
